src/modules/Elsa.Workflows.Sink/Extensions/ModuleExtensions.cs

```diff
@@ -0,0 +1,15 @@
+using System;
+using Elsa.Features.Services;
+using Elsa.Workflows.Sink.Features;
+
+// ReSharper disable once CheckNamespace
+namespace Microsoft.Extensions.DependencyInjection;
+
+public static class ModuleExtensions
+{
+    public static IModule UseWorkflowSink(this IModule module, Action<WorkflowSinkFeature>? configure = default)
```

Comment:
Let's use the plural form `UseWorkflowSinks` - since we're gonna use a feature, within which we may enable one or more sinks.